corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -34,64 +34,90 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
+                          <span> {% trans "Sort By" %} </span><i class="fa fa-sort" aria-hidden="true"></i>
+                        </button>
+                        <ul class="dropdown-menu dropdown-menu-right" role="menu" aria-labelledby="case-list-sort-by">
+                          <%  _.each(headers, function(header, index) { %>
+                            <% if (columnSortable(index)) { %>
+                            <li tabindex="0" class="header-clickable formplayer-request" data-id="<%- index %>" role="menuitem"><a><%= header %></a></li>
+                            <% } %>
+                          <% }); %>
+                        </ul>
+                      </div>
+                    </div>
                   <% } %>
-                  <%  _.each(headers, function(header, index) { %>
-                    <% if (columnVisible(index)) { %>
-                      <% if (columnSortable(index)) { %>
-                        <th class="module-case-list-header header-clickable formplayer-request" data-id="<%- index %>"> <%= header %></th>
-                      <% } else { %>
-                        <th class="module-case-list-header" data-id="<%- index %>"> <%= header %></th>
-                      <% } %>
+                  <section class="js-case-container clearfix list-cell-container-style"></section>
+                <% } else { %>
+                <div class="module-case-list-table-container">
+                  <table class="table module-table module-table-case-list">
+                      <thead>
+                        <tr>
+                          <% if (isMultiSelect && !hasNoItems) { %>
+                            <th class="module-case-list-header"><input type="checkbox" id="select-all-checkbox"/></th>
+                          <% } %>
+                          <%  _.each(headers, function(header, index) { %>
+                            <% if (columnVisible(index)) { %>
+                              <% if (columnSortable(index)) { %>
+                                <th class="module-case-list-header header-clickable formplayer-request" data-id="<%- index %>"> <%= header %></th>
+                              <% } else { %>
+                                <th class="module-case-list-header" data-id="<%- index %>"> <%= header %></th>
+                              <% } %>
+                            <% } %>
+                          <% }); %>
+                        </tr>
+                      </thead>
+
+                    <% if (hasNoItems) { %>
+                      {% include 'formplayer/no_items_text.html' %}
                     <% } %>
-                  <% }); %>
-                </tr>
-              </thead>
+                    <tbody class="wrapper js-case-container">
+                    </tbody>
+                  </table>
+                </div>
+                <% } %>
+              </div>
```

Comment:
This looks like it's closing the same `div` as line 121, and seems to make the pagination controls appear outside the container (normally they're visually part of the case list and share its white background):
![image](https://github.com/dimagi/commcare-hq/assets/100609770/ebfe4d7e-6036-4a87-96ad-2fe7c07563f8)